zoo_frontend/src/util/withAuth.jsx

```diff
@@ -41,17 +39,18 @@ export default (allowedRoles = ['authenticated']) => (WrappedComponent) => {
         return { ...pageProps };
       }
 
-      const api = new Api(c.authToken || '');
-
+      const api = new Api(c.authToken);
       try {
-        await api.validateToken().catch(() => {
+        await api.validateToken().then(async () => {
+          // console.log('pass validation');
+        }, () => {
           if (process.browser) {
-            document.cookie = 'authToken=; path=/';
```

Comment:
I am fairly certain these still need the path because if we set token on something that is not the root path then it will not apply to all routes.

Reply:
I wasn't facing that issue at all. the / path is set by default. I was navigating just fine and verifying that I wasn't calling login or anything and that there were no issues with the cookie. Adding back in to please.